src/Sites/youtube.com/Runtime/Tokenizer.ts

```diff
@@ -74,6 +72,22 @@ export class Tokenizer {
 		ctx.appendChild(span);
 	}
 
+	/**
+	 * Append a text part to the new message context
+	 */
+	addHyperlinkPart(ctx: HTMLSpanElement, text: string): boolean {
```

Comment:
This doesn't  open in a new tab, so if a user clicks a link they redirect to the page and quit the stream

Reply:
Oops, ill fix that